### ISSUE TITLE
Dispatcher service stops updating poller-stats after losing Redis-connection

### ISSUE THREAD

**Issue by ottorei** [librenms/librenms#12707]
We encountered an issue with dispatcher service on a 4 poller distributed setup where the Redis-server was offline for a few hours. This cause 2 / 4 of the pollers to completely fail. For some reason, a few of them did reconnect but the others did not. Maybe the other poller connected at slightly different time when the Redis came back. 

The issue can be fixed by manually restarting the service with systemctl. These are the last lines on log.

![lnms-issue-rc](https://user-images.githubusercontent.com/7031571/113668858-8a672b80-96bb-11eb-9bdc-bc7c9365493e.png)

**Comment by maesbrisa** [comment]
Hi!

We found the same issue in the last version (21.5.1). It turns out that the metrics from the devices lost track until we restart the dispatcher container. We found a temporary solution adding a service that restarts all unhealthy containers affected by redis (we also have to create a new librenms image that allows this feature). [Documentation about docker healthcheck](https://docs.docker.com/engine/reference/builder/#healthcheck).

Hope it helps.
